jetty-server/src/main/java/org/eclipse/jetty/server/handler/ContextHandler.java

```diff
@@ -264,9 +267,8 @@ private ContextHandler(Context context, HandlerContainer parent, String contextP
         _scontext = context == null ? new Context() : context;
         _attributes = new AttributesMap();
         _initParams = new HashMap<>();
-        addAliasCheck(new ApproveNonExistentDirectoryAliases());
         if (File.separatorChar == '/')
-            addAliasCheck(new AllowSymLinkAliasChecker());
+            addAliasCheck(new SymlinkAllowedResourceAliasChecker(this));
```

Comment:
Why only add this for `/`?
Also, the class javadoc seems to be in contrast with what is configured here in code.

Reply:
This is tricky.

Not all FileSystems on Linux or OSX support symlinks.
And some FileSystems on Windows support symlinks.

Basing this default off of File.separatorChar is not wholly accurate.

Reply:
This is pre-existing behaviour which we are maintaining to not break existing deployments.

Reply:
This should be fine, because if a FS does not support symlinks, then this checker will never see a symlink and thus will be a noop